app/components/UI/Modal.js

```diff
@@ -60,7 +60,7 @@ class Modal extends React.Component {
         bg="primaryColor"
         color="primaryText"
       >
-        <Panel.Header p={3}>
+        <Panel.Header pt={3} px={3}>
           <Flex justifyContent="space-between">
             <Box
               css={{ height: '40px', cursor: 'pointer', opacity: 0.6, '&:hover': { opacity: 1 } }}
```

Comment:
Since you are doing an overhaul, could you please extract this into a styled component please?

Reply:
I've cleaned up the Modal component a little.